java-checks-test-sources/src/main/java/checks/InterfaceAsConstantContainerCheck.java

```diff
@@ -3,52 +3,45 @@
 interface InterfaceAsConstantContainerCheck {
 }
 
-interface InterfaceAsConstantContainerCheckB {
-}
-
-interface InterfaceAsConstantContainerCheckC { // Noncompliant [[sc=11;ec=45;secondary=10,11]] {{Move constants defined in this interfaces to another class or enum.}}
+interface InterfaceAsConstantContainerCheckA { // Noncompliant [[sc=11;ec=45;secondary=7,8]] {{Move constants defined in this interfaces to another class or enum.}}
   int a = 0;
   int b = 0;
```

Comment:
The official quote is:
> “an interface which contains no methods, it consists solely of **static fields,** each exporting a constant.”

To me, static can be relaxed, and not only static field should be considered.
In any way, we should add a test with static fields.